openrndr-math/src/main/kotlin/org/openrndr/math/Vector2.kt

```diff
@@ -9,38 +9,31 @@ enum class YPolarity {
     CW_NEGATIVE_Y
 }
 
-/**
- * Double precision vector 2
- */
+/** Double-precision 2D vector. */
 data class Vector2(val x: Double, val y: Double) : Serializable, LinearType<Vector2> {
 
     constructor(x: Double) : this(x, x)
 
-    /**
-     * the Euclidean length of the vector
-     */
+    /** The Euclidean length of the vector. */
     val length: Double
         get() = sqrt(x * x + y * y)
 
-    /**
-     * the squared Euclidean length of the vector
-     */
+    /** The squared Euclidean length of the vector. */
     val squaredLength: Double
         get() = x * x + y * y
 
 
     /**
-     * calculate perpendicular vector
-     * @param polarity polarity, default is [YPolarity.CW_NEGATIVE_Y] (screen space)
+     * Calculates a vector perpendicular to the current one.
+     *
+     * @param polarity The polarity of the new vector, default is [CW_NEGATIVE_Y][YPolarity.CW_NEGATIVE_Y].
      */
     fun perpendicular(polarity: YPolarity = YPolarity.CW_NEGATIVE_Y): Vector2 = when (polarity) {
         YPolarity.CCW_POSITIVE_Y -> Vector2(-y, x)
         YPolarity.CW_NEGATIVE_Y -> Vector2(y, -x)
     }
 
-    /**
-     * a normalized version of the vector
-     */
+    /** Returns a normalized version of the vector. */
```

Comment:
Maybe add "(a unit vector)" ?

This is actually normalized, different from "the normal of a curve at a point t" (which is actually normalized! haha). Crazy human languages...